add-remove.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export function sum(a, b) {
+  return a + b;
+}
```

Comment:
- [ ]  I noticed that this portion of code is redundant; if so, kindly remove it to make your codebase cleaner. 👍